styles.css

```diff
@@ -0,0 +1,1146 @@
+  stroke-dasharray: 40 160;
+}
+
+.ham .middle {
+  stroke-dasharray: 40 142;
+  transform-origin: 50%;
+  transition: transform 400ms;
+}
+
+.ham .bottom {
+  stroke-dasharray: 40 85;
+  transform-origin: 50%;
+  transition: transform 400ms, stroke-dashoffset 400ms;
+}
+
+.ham.active .top {
+  stroke-dashoffset: -64px;
+}
+
+.ham.active .middle {
+  transform: rotate(90deg);
+}
+
+.ham.active .bottom {
+  stroke-dashoffset: -64px;
+}
+
+/* Menu */
+.menu-container {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  height: 100vh;
+  z-index: 1;
+  padding: 30px;
+  background-color: #ec5242;
+}
+
+.menu-close-container {
+  display: flex;
+  justify-content: flex-end;
+  padding: 16px 38px 32px 0;
+}
+
+.close-icon {
+  padding-left: 7%;
+  padding-top: 7%;
+}
+
+ul {
+  padding: 0;
+}
+
+.menu-list > ul {
+  padding-inline-start: 45px;
+}
+
+.menu-list ul li {
+  /* position: relative; */
+  width: 100%;
+  height: calc(100% - 80px);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.menu-list nav {
+  padding-bottom: 70px;
+}
+
+.menu-list {
+  height: calc(100% - 140px);
+  padding-left: 10%;
+  padding-top: 12%;
+}
+
+.menu-item {
+  font-family: "lato", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: #f0eeee;
+  text-decoration: none;
+
+  /* display: none; */
+}
+
+.show-menu {
+  display: flex;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  height: 100vh;
+  z-index: 1;
+  padding: 30px;
+  background-color: #ec5242;
+}
+
+.deskop-version {
+  display: none;
+}
+
+/* ***** */
+.nav-wrapper {
+  height: 70px;
+  display: flex;
+  align-items: center;
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+}
+
+.logo-wrapper {
+  display: none;
+}
+
+.bi {
+  color: orange;
+  font-size: 40px;
+  align-items: center;
+  display: flex;
+  justify-content: center;
+}
+
+.bi-one {
+  color: cornsilk;
+  font-size: 12px;
+}
+
+.science {
+  font-size: 15px;
+  width: 150px;
+  color: #ec5242;
+}
+
+.home-wrapper {
+  background-color: #d3d3d3;
+  color: transparent;
+  padding: 5%;
+}
+
+.bg-wrapper {
+  display: none;
+}
+
+.canpion {
+  display: none;
+}
+
+.hello-wrapper {
+  background-color: #d3d3d3;
+  mix-blend-mode: multiply;
+  height: 150px;
+}
+
+.hello-wrapperh {
+  color: #ec5242;
+  font-size: 30px;
+  font-weight: 600;
+}
+
+.creat-wrapper h1 {
+  font-size: 45px;
+  line-height: 0.9em;
+  font-weight: 900;
+  background-position: left 90%;
+  background-clip: text;
+  color: transparent;
+  color: #ec5242;
+}
+
+.description {
+  color: #272a31;
+}
+
+.pargraph {
+  width: 300px;
+  font-size: 12px;
+}
+
+.date {
+  font-size: 14px;
+  font-weight: 700;
+}
+
+.place {
+  font-size: 12px;
+  font-weight: 200;
+  margin-bottom: 100px;
+}
+
+/* main */
+.mainwrapper {
+  background-color: #000;
+  background-image: url(./image/carbon_fibre.png);
+  height: 960px;
+}
```

Comment:
- [ ] Please kindly make sure your main program design stick as much as possible to the original design,( eg: margin, padding,...)
- [ ] Also make sure there is a divider after the title 😉 

Your design
<img width="365" alt="Screen Shot 2023-01-14 at 04 12 18" src="https://user-images.githubusercontent.com/74501523/212443786-e5e957d9-15db-492d-9501-61ef3ed08997.png">

Original design
<img width="556" alt="Screen Shot 2023-01-14 at 04 13 02" src="https://user-images.githubusercontent.com/74501523/212443813-761f503d-9d9a-48d6-be4f-22f86264092a.png">